app/src/main/java/org/simple/clinic/instantsearch/InstantSearchScreen.kt

```diff
@@ -196,15 +196,15 @@ class InstantSearchScreen(context: Context, attrs: AttributeSet) : ConstraintLay
 
   override fun showAllPatients(patients: List<PatientSearchResult>, facility: Facility) {
     searchResultsView.visibility = View.VISIBLE
-    allPatientsAdapter.submitList(InstantSearchResultsItemType.from(patients, facility))
+    allPatientsAdapter.submitList(InstantSearchResultsItemType.from(patients, facility, ""))
```

Comment:
Instead of passing an empty string that doesn't say what it is. Let's pass a `null` value and use parameter names.